README.md

```diff
@@ -663,11 +665,11 @@ const options = new FileSystemUploadOptions()
     .withUrl('http://localhost:4502/content/dam/target-folder')
     .withBasicAuth('admin:admin');
 
-// upload a single asset and all assets in a given directory
+// upload a single asset and all assets in a given folder
```

Comment:
What is the difference between a folder and a directory?

Reply:
Nothing really, I just updated the readme to consistently use one over the other.